aaad/templates/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <style>
+      .heading {
+        align: center;
+        font: 1em sans-serif;
+        padding-left: 70px;
+        }
+      .formarea {
+        margin: 0 auto;
+        width: 300px;
+        padding: 1em;
+        border: 1px solid #CCC;
+        border-radius: 1em;
+      }
+      div + div {
+        margin-top: 1em;
+      }
+      label {
+        display: inline-block;
+        width: 90px;
+        text-align: right;
+      }
+      .flashes {
+        display: inline-block;
+        width: 90px;
+        text-align: right;
+      }
+      input, textarea, button {
+        font: 1em sans-serif;
+        width: 300px;
+        -moz-box-sizing: border-box;
+             box-sizing: border-box;
+        border: 1px solid #999;
+      }
+      input:focus, textarea:focus {
+        border-color: #000;
+      }
+      textarea {
+        vertical-align: top;
+        height: 5em;
+        resize: vertical;
+      }
+      .button {
+        font: 1em sans-serif;
+      }
```

Comment:
Yes! It was not working with the styles being outside. Turns out it was an nginx configuration for the mime type.